Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -35,7 +58,7 @@ while [ -n "$1" ]; do
   shift
 done
 
-log "Loading defaults and config $ES_CONFIG if any"
+log INFO "Loading defaults and config $ES_CONFIG if any"
```

Comment:
This change has nothing to do with custom EOB/EOP.

Reply:
@mgolosova 

Should it appear in another PR (I'm thinking about to move everything related with log() into another PR and formalize all mentioned connectors)?

Reply:
Hm, separate PR is a great idea.
I always miss the moment when a set of small changes becomes not so small and deserves personal branch...